media-api/test/lib/elasticsearch/ConditionFixtures.scala

```diff
@@ -14,6 +15,12 @@ trait ConditionFixtures {
   val dateMatchCondition = Match(SingleField("adatefield"), DateRange(dateRangeStart, dateRangeEnd))
 
   val hasFieldCondition = Match(HasField, HasValue("foo"))
+
+  val isOwnedPhotoCondition = Match(IsField, IsValue(IsOwnedPhotograph.toString))
+  val isOwnedIllustrationCondition = Match(IsField, IsValue(IsOwnedIllustration.toString))
+  val isOwnedImageCondition = Match(IsField, IsValue(IsOwnedImage.toString))
+  val isInvalidCondition = Match(IsField, IsValue("a-random-string"))
```

Comment:
Entirely not sure if this, but something prevents typing-ahead to select values in `is:` chip. Not a biggie, but just wanted to highlight, @akash1810.